joint_trajectory_controller/include/joint_trajectory_controller/init_joint_trajectory.h

```diff
@@ -314,6 +314,16 @@ Trajectory initJointTrajectory(const trajectory_msgs::JointTrajectory&       msg
                       "s in the past.");
       return Trajectory();
     }
+    else if ( // If the first point is at time zero and no start time is set in the header, skip it silently
+              msg.points.begin()->time_from_start == ros::Duration(0.) &&
```

Comment:
Please chnage this, there is no need to call the constructor and the comparison operator instead of using the function that was implemented for this purpose.